docs/source/user_guide/pipelines.md

```diff
@@ -104,7 +104,17 @@ elyra-pipeline submit elyra-pipelines/demo-heterogeneous.pipeline \
 
 The `runtime-config`  should be a valid [runtime configuration](/user_guide/runtime-conf.md).
 
+#### Detecting the runtime from a component
```

Comment:
We need to introduce the term `component` because afaik we haven't used it prior to 2.3.x in the context of pipeline artifacts. This is also needed because the Pipelines documentation topic starts with "Elyra utilizes its canvas component ", which is semantically completely different and can lead to confusion.

Reply:
Since we reference "nodes" I use that.   Thanks.